drivers/hmis/app/models/hmis/hud/processors/current_living_situation_processor.rb

```diff
@@ -6,6 +6,37 @@
 
 module Hmis::Hud::Processors
   class CurrentLivingSituationProcessor < Base
+    def process(field, value)
+      attribute_name = ar_attribute_name(field)
+      current_living_situation = @processor.send(factory_name)
+
+      return super(field, value) unless attribute_name == 'verified_by_project_id'
+
+      verified_by_project_id = attribute_value_for_enum(graphql_enum(field), value)
+
+      if verified_by_project_id
+        # We collect project ID onto the non-HUD field `verified_by_project_id`,
+        # then copy the corresponding project name onto the HUD field `VerifiedBy` (aka `verified_by`).
+        project_name = Hmis::Hud::Project.find(verified_by_project_id).name
```

Comment:
It might help for future debugging to raise something more explicit if the project id isn't found:

```
project = Hmis::Hud::Project.find_by(id: verified_by_project_id)
raise 'CLS Verified By project not found' unless project
```

Reply:
Actually, I did encounter this error when editing the date on a "migrated-in" CLS record. In that situation the `verified_by_project_id` is submitted as whatever value was in `VerifiedBy`. Maybe this would be safer:

```
verified_by_project = Hmis::Hud::Project.find_by(id: verified_by_project_id) if verified_by_project_id

if verified_by_project
   # a project was selected, so assign attributes verified_by_project_id and  verified_by
elsif current_living_situation.verified_by_project_id
  # a project was previously selected and is now cleared. null out values
end
```
Thoughts?